src/screens/HomeScreenV2/sections/AnnouncementSection.tsx

```
@@ -100,7 +106,7 @@ export default function AnnouncementSection({
               <AnnouncementSlot
                 key={item.slot}
                 style={{top: item.slot * ITEM_HEIGHT}}>
-                <AnnouncementItem announcement={item.announcement} />
+                <AnnouncementItem announcement={item.announcement} trackView={item.slot === renderCenter} />
```

ESLint

[error] Replace `·announcement={item.announcement}·trackView={item.slot·===·renderCenter}` with `⏎··················announcement={item.announcement}⏎··················trackView={item.slot·===·renderCenter}⏎···············`

(prettier/prettier)